src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,36 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.2.0
+
+### New Feature
+
+- Support for self-signed and private CA with Docker
+- Buildkit port is now configurable (default 443)
+- Add option for sharing namespace with users within organization
+- UI pagination support for GitHub branches list
+- Add permanently user deletion command
+- Allow to scale down to zero nodes if no user pods are running
+- Populate oauth errors to the user
+- Added new helm values to set PSP and AppArmor
+- Update okteto cli version to 2.9.0
+
+### Bug Fix
+
+- Don't show warning on k8s 1.23.x
+- Frontend nginx CVE-2022-40303/CVE-2022-40304
+- Fix loader icon regression
+- Fix in license display
+- Update text in error when using token-based auth
+- Persist selfsigned wildcard certificate
+- Update daemon image
+- Allow pod to have its own image pull secret
+- Fix "Uncaught ReferenceError: $RefreshReg$ is not defined"
+- Add registry hostname to installer /etc/hosts
+- Fix user deletion when there is no owner among the namespace members
+- Fix pending to sleep label
+- Fix issues showing virtual service endpoints
```

Comment:
Rephrase to be about the user benefit ("virtual services are now correctly displayed in the dashboard " or something like that)